### PR TITLE
TMT demo

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

**Comment by martinpitt** [comment]
TMT infra is [buggy with multiple package:s](https://console-testing-farm.apps.ci.centos.org/pipeline/bad872c7-fb5a-4578-ad8b-6ffd5ae0b96c), I filed that as https://pagure.io/fedora-ci/general/issue/170

**Comment by martinpitt** [comment]
Now it [failed](https://console-testing-farm.apps.ci.centos.org/pipeline/e9b8cce0-f4be-4249-9ad9-ad622450622d) because `make ci` is apparently not running in an actual git repo checkout. Adding some debug output.

File: plans/unit-tests.fmf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+summary: Run unit tests
+prepare:
+  how: shell
+  script: dnf install -y python3-pylint python3-gobject-base make
+execute:
+    # HACK: not invoked in right directory; it's being fixed with the packit migration
+    script: [ -d git-source ] && cd git-source || true; make ci
```
Comment (thrix) [inline plans/unit-tests.fmf:7]: @martinpitt it is a yaml problem, this should fix it:

```suggestion
    script: "[ -d git-source ] && cd git-source || true; make ci"
```

Note that fmf should always be a valid yaml

Comment (thrix) [reply plans/unit-tests.fmf:7]: the result is now:

```
[20:26:03] [+] [cruncher] [/testik/test] [execute] Running shell commands
# [ -d git-source ] && cd git-source || true; make ci
make: *** No rule to make target 'ci'.  Stop.
```
I guess that is something finally relevant

Comment (thrix) [reply plans/unit-tests.fmf:7]: ah actually it is `~/git-source` :/

Comment (thrix) [reply plans/unit-tests.fmf:7]: I deleted two additional comments as they were wrong with the above I now see what I would expect:

```
# free -t -h; echo "nproc $(nproc)"; ls -l /dev/kvm || true; make ci
              total        used        free      shared  buff/cache   available
Mem:          1.8Gi        84Mi       1.2Gi       2.0Mi       500Mi       1.5Gi
Swap:            0B          0B          0B
Total:        1.8Gi        84Mi       1.2Gi
nproc 1
crw-rw-rw-. 1 root kvm 10, 232 Nov 10 14:12 /dev/kvm
make: *** No rule to make target 'ci'.  Stop.
[20:38:00] [E] [cruncher] [/plans/unit-tests] Command 'cd ~/; free -t -h; echo "nproc $(nproc)"; ls -l /dev/kvm || true; make ci' failed with return code '2'
```

I looked at AWS, and no nested virt there :(

Comment (martinpitt) [reply plans/unit-tests.fmf]: Thanks! This was mostly just a probe how the environment looks like. I also fixed the `script:` thing.

**Comment by martinpitt** [comment]
OK, pylint is too old on F31/32 for the new raise-missing-from keyword, but that's now totally uninteresting from an infrastructure POV. This generally works fine now.

**Pull request closed without merging**